platformio.ini

```diff
@@ -40,24 +51,22 @@ platform = ststm32
 board = custom_stm32
 build_flags =
   ${env.build_flags}
-  -fstrict-volatile-bitfields
+  ${respiraworks.stm32_build_flags}
   -DBARE_STM32
-  -mfpu=fpv4-sp-d16
-  -mfloat-abi=hard
-  -Wl,-Map,stm32.map
-  -Wl,-u,vectors
-  -Wl,-u,_init
 board_build.ldscript = boards/stm32_ldscript.ld
-build_unflags = -std=gnu11 -std=gnu++14
 extra_scripts = pre:boards/stm32_scripts.py
 src_filter = ${env.src_filter} -<test/> -<src_test/>
 
+# STM32 with DMA-based communication.  Will eventually become main build.
 [env:stm32-test]
```

Comment:
More like "integration testing experiment", I don't think this will become the main build.

Reply:
Thanks, fixed.